### PR TITLE
CHIRP 20150702 (new formula)

### PR DESCRIPTION
**Pull request by johanoskarsson**
CHIRP is a tool for programming an amateur radio.
They rely on daily builds instead of stable releases, hence 20150702.

### CODE REVIEW

File: Library/Formula/chirp.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  url "http://trac.chirp.danplanet.com/chirp_daily/daily-20150702/chirp-daily-20150702.tar.gz"
+  sha256 "41a5ae113bea27a284675ad5d95b45e4cd90f76ea2cd29050396d55bab5be433"
+
+  depends_on "python"
+  depends_on "pygtk"
+  depends_on "libxml2" => "with-python"
+
+  def install
+    prefix.install Dir["*"]
+
```
Comment (bfontaine) [inline Library/Formula/chirp.rb]: You can drop this empty line.

File: Library/Formula/chirp.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    bin.install_symlink "#{prefix}/chirpw"
+  end
```
Comment (bfontaine) [inline Library/Formula/chirp.rb]: Please add a `test do` block.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by DomT4** [comment]
Test failing:

```
Error: chirp: failed
Expected /CHIRP/ to match "No handlers could be found for logger \"chirp.platform\"\n".
```

**Comment by dunn** [comment]
Is every daily build stable? If not, maybe this should go in the devel-only tap.

**Comment by DomT4** [comment]
> CHIRP is distributed as a series of automatically-generated builds. Any time we make a change to CHIRP, a build is created for it the next day. Thus, CHIRP is versioned by the date on which it was created, which makes it easy to determine if you have an older build. We don't put experimental things into CHIRP before they are ready, except where specifically called out with a warning. Thus, you do not need to worry about finding a stable version to run. You should always be on the latest build available.

I'm not sure this is necessarily a good fit for Homebrew if it's something most users are going to expect to see updated *daily*.

File: Library/Formula/chirp.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class Chirp < Formula
+  desc "CHIRP is a free, open-source tool for programming your amateur radio."
+  homepage "http://chirp.danplanet.com/"
+  url "http://trac.chirp.danplanet.com/chirp_daily/daily-20150702/chirp-daily-20150702.tar.gz"
+  sha256 "41a5ae113bea27a284675ad5d95b45e4cd90f76ea2cd29050396d55bab5be433"
+
+  depends_on "python"
```
Comment (DomT4) [inline Library/Formula/chirp.rb]: We don't hardcode our Python.

```ruby
depends_on :python if MacOS.version <= :snow_leopard
```

File: Library/Formula/chirp.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "pygtk"
+  depends_on "libxml2" => "with-python"
+
+  def install
+    prefix.install Dir["*"]
```
Comment (DomT4) [inline Library/Formula/chirp.rb]: Use `libexec.install` & then `bin.install_symlink libexec/"chirpw"`.

**Comment by DomT4** [comment]
Going to pass on this because of the daily-update expectation as noted in https://github.com/Homebrew/homebrew/pull/41686#issuecomment-145303503. Just not sure that makes it a good fit for Homebrew, or provides a desirable user experience.

If they ever move onto more stable releases which don't expect to be updated daily, we'd certainly be happy to review this again. Thanks for the submission :heart:

**Pull request closed without merging**